perceval/serialization/_circuit_serialization.py

```diff
@@ -138,6 +139,51 @@ def _serialize(self, barrier: comp.Barrier):
         pb_barrier.visible = barrier.visible
         self._pb.barrier.CopyFrom(pb_barrier)
 
+    @dispatch(FFConfigurator)
+    def _serialize(self, ffconfigurator: FFConfigurator):
+        pb_ffc = pb.FFConfigurator()
+        pb_ffc.name = ffconfigurator.name
+        pb_ffc.offset = ffconfigurator._offset
+        pb_ffc.block_circuit_size = ffconfigurator._blocked_circuit_size
```

Comment:
I'm not sure we want to serialize this, this is just a boolean that indicates if the Configurator has been placed in an experiment.
If we serialize and deserialize it on its own, we lose the reference of the deserialized object in said experiment, so we don't need to block the size. If we serialize it in an experiment and we deserialize the experiment, this boolean will be set to True when we add the deserialized Configurator to the deserializing Experiment.
The only case where it could be True without an Experiment is if a user calls this method themselves, but I assume this is never the case

Reply:
As `block_circuit_size()` is a public method from the `FFConfigurator` class, I was expecting to have to serialize it, as we can't deduce this boolean value from the context where it's being used.